endpoint/domain_filter.go

```diff
@@ -146,35 +149,32 @@ func matchRegex(regex *regexp.Regexp, negativeRegex *regexp.Regexp, domain strin
 
 // MatchParent checks wether DomainFilter matches a given parent domain.
 func (df DomainFilter) MatchParent(domain string) bool {
-	if !df.IsConfigured() {
```

Comment:
Why did you modify `IsConfigured` below if you are removing it here?

Reply:
`IsConfigured` is still consumed at other call sites.